views/js/e2e/runner/_setup/setupCommands.js

```diff
@@ -92,5 +95,43 @@ Cypress.Commands.add('setDeliveryForGuests', () => {
 
     // Wait until save happened properly
     // Not ideal but these requests have to be waited in this order upon delivery save
-    cy.wait(['@editDelivery', '@getData','@editDelivery', '@getData', '@editDelivery' ]);
-});
\ No newline at end of file
+    cy.wait(['@editDelivery', '@getData','@editDelivery', '@getData', '@editDelivery', '@getData' ]);
+});
+
+Cypress.Commands.add('startTest', () => {
```

Comment:
I think it would be a nice idea in general to start every `Cypress.Commands.add()` function with a `cy.log('COMMAND: myCommand')` statement. It makes it easier to debug the tests when you are writing them and when they are failing.

We did this in the `tao-core` E2E tests: https://github.com/oat-sa/tao-core/blob/master/views/js/e2e/resourceSelector/resourceTree.js#L170

Reply:
Agree

Reply:
yes.